server/server_test.go

```diff
@@ -722,3 +714,117 @@ func TestHTTPServer_deleteClusters_BadClusterName(t *testing.T) {
 		Body:       `{"status": "Error during parsing param 'cluster' with value 'aaaa'. Error: 'invalid UUID length: 4'"}`,
 	})
 }
+
+func TestHTTPServer_SaveDisableFeedback(t *testing.T) {
+	mockStorage, closer := helpers.MustGetMockStorage(t, true)
+	defer closer()
+
+	err := mockStorage.WriteReportForCluster(
+		testdata.OrgID, testdata.ClusterName, testdata.Report3Rules, testdata.LastCheckedAt, testdata.KafkaOffset,
+	)
+	helpers.FailOnError(t, err)
+
+	const expectedFeedback = "user's feedback"
+
+	helpers.AssertAPIRequest(t, mockStorage, nil, &helpers.APIRequest{
+		Method:       http.MethodPost,
+		Endpoint:     server.DisableRuleFeedbackEndpoint,
+		EndpointArgs: []interface{}{testdata.ClusterName, testdata.Rule1ID, testdata.UserID},
+		Body:         fmt.Sprintf(`{"message": "%v"}`, expectedFeedback),
+	}, &helpers.APIResponse{
+		StatusCode: http.StatusOK,
+		Body:       `{"message":"user's feedback", "status":"ok"}`,
+	})
+
+	feedback, err := mockStorage.GetUserFeedbackOnRuleDisable(testdata.ClusterName, testdata.Rule1ID, testdata.UserID)
+	helpers.FailOnError(t, err)
+
+	assert.Equal(t, expectedFeedback, feedback.Message)
+}
+
+func TestHTTPServer_SaveDisableFeedback_Error(t *testing.T) {
+	t.Run("BadClusterName", func(t *testing.T) {
```

Comment:
Usually for each test we write own function, so to be consistent I suggest split it.

Reply:
Why to copy paste the name each time when we can use a function as a namespace?

Reply:
In consistency sake. @Sergey1011010 @tisnik do we want to change test style?

Reply:
@bond95 @Sergey1011010 what's the problem?

Reply:
@tisnik inconsistency with other tests.

Reply:
+1 @bond95

Reply:
@tisnik no problems, splitted to multiple functions

Reply:
TY @Sergey1011010